### ISSUE TITLE
Interested in Collaborating on Sonics

### ISSUE THREAD

**Issue by moadel321** [electronicalias/np2#1]
Hello Philip

I came across your comments on the Seasme fine-tuning issue, which led me to https://sonics.app/ 

I ended up using other code for a full fine-tune as I needed to adapt Seasme for a new language..... the results were okay, at best.... need more data 

This brings me back to Sonics. I believe this is a great idea. I've been training speech models for the past year and a half, and data is always, always the bottleneck.

I'd be genuinely interested in collaborating or contributing to the project. If you're open to it, I'd love to connect and explore how I might be able to help develop and enhance the platform

I'm a 31-year-old developer from Egypt, My thesis is that AI voice agents have the potential to be the next frontier in human-computer interaction, and I'm passionate about building something that can pass the Turing test. To do that, we have quite a big gap in publicly available datasets 

Apologies for using GitHub Issues, but I couldn't find any contact methods 

Cheers,
Mo

**Comment by electronicalias** [comment]
Hey Mo,

Thank you for reaching out, I wasn't expecting it. I am not the worlds best developer, but I've certainly taken an interest in this field. I'm glad you did use an issue and that I saw the notification. 

The site is currently based on a Lovable.dev front end, with an AWS back end. I first got the idea when I was playing around with voice to support the SaaS platform I have built that performs testing of contact centre voice journeys. We currently use a large selection of API based voice generators and have played around with pre-generated and realtime in call voice.

The real advantage this might give our platform, is voices that can have 'accents'. For example, you can apply many different colloquial accents to English - just in the UK, there are vast differences between Birmingham and Liverpool for example. That ultimately got me finding YouTube videos to take audio from to research it - but never publishing or using these commercially since we wouldn't have permission.

Like you I discovered that manually doing all of this work to fine tune any one of the current models like CSM-1b or Orpheus, was tedious at best, with mixed results. As it stands, the platform can now take source audio, transcribe it, use the information in the transcript to split the audio on sentence boundaries and create a 'dataset' which gets stored in Amazon's S3. The plan was to be able to automate processing this further and fine-tuning models, providing the user with a way to download the results at the end - or maybe even test generate audio.

I would definitely think there is benefit to us collaborating in some way, since this is such early stage and as you can see there is a path to it becoming SaaS driven, I think it would be great to share ideas.

I cannot devote my full time to it currently, but I would really appreciate a conversation at some point in the next week?

Thanks,
Phil

**Comment by moadel321** [comment]
Hello Phil,

Great to hear back from you!

I've reached the same conclusion as yours regarding the lack of datasets for various accents and languages. There's significant demand for a platform or tool that can automatically generate or extract speech datasets.

As far as I understand, this can be done in two ways:

- Purely synthetic: Generating audio from TTS models using a wide variety of voices.
- Extraction approach: Extracting audio from platforms like YouTube, followed by steps such as diarization, clipping, quality estimation, and potentially pre-processing to improve the audio quality or remove bad samples.

I've found some promising results using pre-trained quality estimators like MetricGAN+
I've also been experimenting with using a multimodal LLM as a judge—for instance, feeding audio clips to Gemini 2.5 Flash and asking it to assess their quality. Not very cost-efficient but points out flaws in samples that normal quality estimators don't usuakky catch 

I believe there's considerable opportunity here, especially given the growing demand for data in under-resourced accents and languages.

I'd be happy to chat this week. I'm available anytime Wednesday, Thursday, or Friday, I'm on GMT+3

If that doesn't fit your schedule, then let's schedule early next week

My email is m.adelomar1@gmail.com

Please let me know what works best for you


Cheers,
Mo
